src/tests/JIT/opt/Or/IntOr.cs

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace CodeGenTests
+{
+    class IntOr
+    {
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        static void SideEffect()
+        {
+        }
+
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        public static bool Test_UInt32_UInt32_CastByte_Or(uint x, uint y)
+        {
+            // X64-NOT: movzx
+
+            // We expect 'or r8, r8'.
+            // X64: or [[REG0:[a-z]+[l|b]]], [[REG1:[a-z]+[l|b]]]
```

Comment:
If you're not going to refer to `REG0`/`REG1` again, then the `{{<regex>}}` notation is less cluttered.